src/python/pants/backend/project_info/tasks/idea_plugin_gen.py

```diff
@@ -119,7 +120,7 @@ def generate_project(self):
     configured_project = TemplateData(
       root_dir=get_buildroot(),
       outdir=outdir,
-      git_root=scm.worktree,
+      git_root=scm.worktree if scm else None,
```

Comment:
This is necessary for V2 + when users don't have git/SCM set up. The `.ipr` file simply won't include this info if set to `None`.